### PR TITLE
feat: added responsive images

### PR DESCRIPTION
**Pull request by sks-21**
Added **responsive image** component
Includes

- Square image
- Round image

### CODE REVIEW

File: components/image/image.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    <script src="https://kit.fontawesome.com/0a41da8839.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/components/image/image.css" />
+</head>
+
+<body>
+    <!-- || Container  -->
+    <div class="container">
+
+
+        <!-- || Navbar starts -->
```
Comment (Avin008) [inline components/image/image.html:18]: nice comments

File: components/image/image.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                </ul>
+            </div>
+        </nav>
+
+        <!-- || Side Menu starts   -->
+        <aside class="menu flx-column flx-main-center">
+            <ul class="menu-list">
+                <li class="menu-list-title">Getting Started</li>
+                <li>
+                    <a href="">Installation</a>
```
Comment (Avin008) [inline components/image/image.html:41]: use # to block reloads

**Review by Avin008 — APPROVED** [review]
code looks awesome. good to merge

**Review by shaksham08 — APPROVED** [review]
Looks good

**Pull request merged**